### ISSUE TITLE
[Bug]: XLIFF import issues

### ISSUE THREAD

**Issue by valentin-** [pimcore/pimcore#11724]
### Expected behavior

I came across several issues using the xliff importer. I'm creating a PR for those bugs i fixed locally.

When translating object relations, the items are created in the default language. If possible they should get translated as well (use document for en, not de).

Pimcore 10.3.3

### Actual behavior

The import file only contains a single field.

```
<?xml version="1.0" encoding="UTF-8"?>
<xliff version="1.2">
  <file original="object-8" source-language="de" target-language="en" datatype="html" tool="pimcore" category="object">
    <header />
    <body>
      <trans-unit id="localizedfield~-~secondColHeadline">
        <source xml:lang="de">
          <![CDATA[Unsere Produkte]]>
        </source>
        <target state="translated">
          <![CDATA[Our products]]>
        </target>
      </trans-unit>
    </body>
  </file>
</xliff>
```

This also updates the links for `secondColLinks`:

![image](https://user-images.githubusercontent.com/90033/159288773-c41b92c2-c008-4ead-a358-2b97c46d4ed7.png)

In EN only the headline has been imported, links are empty:

![image](https://user-images.githubusercontent.com/90033/159288805-43db6ded-bace-4031-9852-4477c16cf9d0.png)

The links are only present in the object_relations Table, the object itself only contains one link.

Relation:
![image](https://user-images.githubusercontent.com/90033/159289181-59661a0c-4983-4a23-a0e7-57f15169a586.png)

Data Object:
![image](https://user-images.githubusercontent.com/90033/159289318-f95db37c-6ddd-4c72-b9ec-6d81af29471f.png)

---

I have no clue why/where the links are created. Is this working as intended?

### Steps to reproduce

-

**Comment by dvesh3** [comment]
@valentin- Sorry, the description is not quite understandable to me. Could you please share steps to reproduce the issue? thanks!

**Comment by dvesh3** [comment]
@valentin- 🏓

**Comment by valentin-** [comment]
@dvesh3 sorry I forgot to reply.

- Create a document tree (DE)
- Create a data object with settings and apply it as property to the document root
- Clone tree for document translation (EN)
- Create xliff export of the new tree
- Import xliff file into the new tree

Only the headline will get imported in the desired language (inside the settings data object), object relations are applied using the default record in the default language - not the translated object/document in the target language.

I hope this helps to understand the issue better.

**Comment by dvesh3** [comment]
@valentin- XLIFF import does not handle relation data types. only the localized text types are supported. In your case, you need to manually update the relations in data objects.

**Comment by valentin-** [comment]
I understand but in my case there are relations created, even though there is only the headline present in the xliff file. Could you please double check the behaviour?

**Comment by brusch** [comment]
Is inheritance enabled in your class definition? Can you share the definition? We need a reproducible case in order to check ...

**Comment by valentin-** [comment]
@brusch inheritance is not enabled.
@dvesh3 attached you find a full example (object definition, export file, import file) - [files.zip](https://github.com/pimcore/pimcore/files/8479859/files.zip)

Before importing there was only one doucment selected. Also the subheader and text for DE are overwritten with values from EN. 

![image](https://user-images.githubusercontent.com/90033/163140213-8367b4b6-7c83-4809-bded-a3839055d934.png)

**Comment by dvesh3** [comment]
@valentin- could you please confirm if https://github.com/pimcore/pimcore/pull/11905 fixes the issues?

**Comment by valentin-** [comment]
@dvesh3 looks good to me. I've cleaned the data and reran the import.

DE:
![image](https://user-images.githubusercontent.com/90033/163188308-2b3c6360-487a-4463-b35c-0a95461faa14.png)

EN:
![image](https://user-images.githubusercontent.com/90033/163188354-10db24e5-1fa8-4f9b-8317-1d1702dca2dd.png)

**Comment by dvesh3** [comment]
Fixed by #11905
